plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,751 @@
+      - If V(true), the job will be run as soon as possible if it was missed while the scheduler was not running.
+    type: bool
+  schedule:
+    description:
+      - Backup schedule. The format is a subset of `systemd` calendar events.
+    type: str
+  script:
+    description:
+      - Use specified hook script.
+    type: str
+  starttime:
+    description:
+      - Job Start time.
+    type: str
+  stdexcludes:
+    description:
+      - Exclude temporary files and logs.
+    type: bool
+  stop:
+    description:
+      - Stop running backup jobs on this host.
+    type: bool
+  stopwait:
+    description:
+      - Maximal time to wait until a guest system is stopped (minutes).
+    type: int
+  storage:
+    description:
+      - Store resulting file to this storage.
+    type: str
+  tmpdir:
+    description:
+      - Store temporary files to specified directory.
+    type: str
+  vmid:
+    description:
+      - The ID of the guest system you want to backup.
+      - Can not be use with vmid and pool in same job
+    type: str
+  zstd:
+    description:
+      - Zstd threads. V(N=0) uses half of the available cores,
+      - if V(N) is set to a value bigger than V(0), V(N) is used as thread count.
+    type: int
+
+extends_documentation_fragment:
+  - community.general.proxmox.actiongroup_proxmox
+  - community.general.proxmox.documentation
+  - community.general.attributes
+'''
+
+EXAMPLES = '''
+- name: List all backup jobs
+  community.general.proxmox_backup:
+    api_host: "node1"
+    api_user: user@realm
+    api_password: password
+    validate_certs: false
+    state: list
+  register: backup_result
+
+- name: Show current backup job
+  ansible.builtin.debug:
+    var: backup_result
+
+- name: Create backup with id backup-20bad73a-d245
+  community.general.proxmox_backup:
+    api_host: "node1"
+    api_token_id: "token_id"
+    api_token_secret: "token_secret"
+    validate_certs: false
+    id: "backup-20bad73a-d245"
+    vmid: "103"
+    mode: "snapshot"
+    mailnotification: "always"
+    mailto: "preprod@idnow.io"
+    repeat_missed: 0
+    enabled: 1
+    prune_backups:
+        keep_yearly: "6"
+        keep_weekly: "5"
+        keep_hourly: "4"
+        keep_daily: "2"
+        keep_last: "1"
+        keep_monthly: "3"
+    storage: "backup-idcheck-preprod-0"
+    schedule: "*-*-* 22:00:00"
+    state: present
+
+- name: Delete backup job
+  community.general.proxmox_backup:
+    api_host: "node1"
+    api_token_id: "token_id"
+    api_token_secret: "token_secret"
+    validate_certs: false
+    id: "backup-20bad73a-d245"
+    state: absent
+
+- name: Update backup with id backup-20bad73a-d245 (Change VM ID backuped)
+  community.general.proxmox_backup:
+    api_host: "node1"
+    api_token_id: "token_id"
+    api_token_secret: "token_secret"
+    validate_certs: false
+    id: "backup-20bad73a-d245"
+    vmid: "111"
+    state: present
+'''
+
+RETURN = '''
+proxmox_backup:
+    description: List of Proxmox VE backup.
```

Comment:
```suggestion
    description: List of Proxmox VE backups.
```

Reply:
Fix in 53df0504d36dc7ee70c350ae3467cca413e0a8b0